stle.css

```diff
@@ -448,3 +468,289 @@ header {
   color: #5b5957;
   background: #f4f1f0;
 }
+
+@media (min-width: 1180px) {
+  * {
+    margin: 0;
+    padding: 0;
+  }
+
+  .nav-bar {
+    margin: 0;
+    font-family: "Inter", sans-serif;
+    font-size: 15px;
+    padding: 0 80px;
+  }
+
+  .nav-bar .menu img {
+    display: none;
+  }
+
+  .nav-bar .menu-text {
+    display: block;
+  }
+
+  .nav-bar .menu-text img {
+    display: block;
+  }
+
+  .nav-bar .menu-text ul {
+    font-family: "Inter", sans-serif;
+    font-size: 15px;
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .nav-bar .menu-text ul li {
+    padding: 0 30px;
+  }
+
```

Comment:
Please ensure that you must stick to the design as much as possible (e.g., font, colors, images, buttons, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/154674848-163ac275-97ea-42a3-92ea-89fb13e102fa.png) | ![image](https://user-images.githubusercontent.com/10332499/154674768-ca1e1907-dfc2-4bb2-af3e-e245283629fa.png)

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/154675220-adcb86d2-35ba-4dbd-b4ac-d07c9788b9c1.png) | ![image](https://user-images.githubusercontent.com/10332499/154675067-ab13a356-dc27-4ecc-bbc2-98ce613462b1.png)

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/154675794-b58c7918-ecc9-4759-b1e2-7159d5235894.png) | ![image](https://user-images.githubusercontent.com/10332499/154675451-9262959d-f8cc-4023-b5a0-464edca25563.png)

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/154677230-0f346ec6-6e70-4ad7-b931-60415a761a41.png) | ![image](https://user-images.githubusercontent.com/10332499/154677546-2a7371ee-b76e-49d6-aff5-cd390d7e8e0c.png)

Please ensure that the message on the hero `page` is on `four` lines like the design and the `social icons` are properly align-center as in the design.